packages/patternfly-4/react-core/src/components/Select/Select.d.ts

```diff
@@ -0,0 +1,16 @@
+import { HTMLProps, FormEvent } from 'react';
+import { Omit } from '../../typeUtils';
+
+export interface SelectProps extends Omit<HTMLProps<HTMLOptionElement>> {
+  isExpanded?: boolean;
+  onSelect: Function;
+  onToggle: Function;
+  placeholderText?: string;
+  selectOptions?: array;
+  selections?: string;
+  variant?: string;
+}
+
+declare const Select: React.SFC<SelectProps>;
```

Comment:
`React.FunctionComponent` or `React.FC` those are the new typings. I know people are going through and updating the old `SFC` references now.